dashboards/google-application-integration/integration-executions.json

```diff
@@ -0,0 +1,502 @@
+              }
+            }
+          },
+          "width": 7,
+          "xPos": 0,
+          "yPos": 6
+        },
+        {
+          "height": 4,
+          "widget": {
+            "timeSeriesTable": {
+              "columnSettings": [
+                {
+                  "column": "status",
+                  "visible": true
+                },
+                {
+                  "column": "integration",
+                  "visible": true
+                }
+              ],
+              "dataSets": [
+                {
+                  "minAlignmentPeriod": "86400s",
+                  "timeSeriesQuery": {
+                    "timeSeriesFilter": {
+                      "aggregation": {
+                        "alignmentPeriod": "86400s",
+                        "crossSeriesReducer": "REDUCE_SUM",
+                        "groupByFields": [
+                          "metric.label.\"status\"",
+                          "resource.label.\"integration\""
+                        ],
+                        "perSeriesAligner": "ALIGN_DELTA"
+                      },
+                      "filter": "metric.type=\"integrations.googleapis.com/integration/execution_count\" resource.type=\"integrations.googleapis.com/IntegrationVersion\" metric.label.\"status\"!=\"SUCCEEDED\"",
+                      "pickTimeSeriesFilter": {
+                        "direction": "TOP",
+                        "numTimeSeries": 5,
+                        "rankingMethod": "METHOD_MEAN"
+                      },
+                      "secondaryAggregation": {
+                        "alignmentPeriod": "60s",
+                        "crossSeriesReducer": "REDUCE_NONE",
+                        "perSeriesAligner": "ALIGN_NONE"
+                      }
+                    }
+                  }
+                }
+              ],
+              "metricVisualization": "BAR"
+            },
+            "title": "Top 5 Integration Executions Failed/Cancelled (daily count)"
+          },
+          "width": 5,
+          "xPos": 7,
+          "yPos": 6
+        },
+        {
+          "height": 2,
+          "widget": {
+            "scorecard": {
+              "blankView": {},
+              "thresholds": [],
+              "timeSeriesQuery": {
+                "apiSource": "DEFAULT_CLOUD",
+                "outputFullDuration": true,
+                "timeSeriesFilter": {
+                  "aggregation": {
+                    "alignmentPeriod": "60s",
+                    "crossSeriesReducer": "REDUCE_SUM",
+                    "groupByFields": [],
+                    "perSeriesAligner": "ALIGN_SUM"
+                  },
+                  "filter": "metric.type=\"integrations.googleapis.com/integration/execution_count\" resource.type=\"integrations.googleapis.com/IntegrationVersion\" metric.label.\"status\"=\"SUCCEEDED\""
+                }
+              }
+            },
+            "title": "Total Integrations Succeeded"
```

Comment:
Does this title (as well as the next 3) need to mention _executions_, or is this a count of integrations?

Reply:
Yes, this is needed, since it is count of Integrations.